src/SFA.DAS.Data.DatabaseTests/Features/Views/LevyDetails.feature

```diff
@@ -0,0 +1,19 @@
+﻿Feature: LevyDetails
+	In order to make sure that i am presenting right data
+	As a Test Analyst
+	I want to be told the views are working as expected
```

Comment:
Could we change the wording here (and ultimately the test) to capture the intent from the point of view of the user (i.e. consumers of the view)? A "test analyst" isn't an end-consumer of the data product...

The levy details view is there to allow end-users to inspect monthly funds coming into individual AS accounts.